app/models/services/locators/doi_paper_locator.rb

```diff
@@ -6,30 +6,79 @@ def column
   def find_paper
     return super if super
 
+    paper = Paper.new
+
+    crossref = Crossref.new
+    if (result = crossref.get_doi_metadata(self.locator_id))
```

Comment:
Let's get rid of these if statements so that the whole process just explodes if the `get_doi_metadata` method fails.